src/routes/user/getUsers.ts

```diff
@@ -4,14 +4,12 @@ import { knexController } from '../../database/database';
 
 import { User, userAdminResponse } from '../../types/user.types';
 
-const fetchAllResponse = Type.Array(userAdminResponse);
-
 const schema = {
   description: 'Fetch all users',
   summary: 'Fetch all users',
   tags: ['User'],
   response: {
-    200: fetchAllResponse,
+    200: Type.Array(userAdminResponse),
```

Comment:
```suggestion
    200: Type.Array(userResponse),
```
Tuossa ei ole mitään admin-only kamaa mukana (ei sillä, noi tyypit on muutenkin identtisiä)

Reply:
Poistin koko tyypin turhana